_test/test_sqw/test_PixelData.m

```diff
@@ -1100,7 +1100,7 @@ function test_append_returns_editied_pix_if_nargout_eq_1(obj)
             ref_range = [min(range1(1,:),range2(1,:));...
                 max(range1(2,:),range2(2,:))];
             out_pix = pix.append(pix_to_append);
-            % img_range, stored in the file is different from
+            % img_db_range, stored in the file is different from
```

Comment:
Is this an overzealous sed?

Reply:
no -- its intentional. the variable I mention is img_db_range and it is different from img_range.